fetch.bs

```diff
@@ -3261,6 +3598,9 @@ steps:
      <li>If <var>cookies</var> is not the empty string, append
      `<code>Cookie</code>`/<var>cookies</var> to <var>httpRequest</var>'s
      <a for=request>header list</a>.
+
+     <li><p>If the user agent supports <a for=/>Token Binding</a>,
```

Comment:
I'd drop this, here and elsewhere: user agents that don't support token binding won't set the header because they're not compliant with the spec. The spec shouldn't carve out normative requirements like this.